frontend/src/components/Reviews/ReviewForm.vue

```diff
@@ -9,6 +9,9 @@
       <span class="required">*</span>
       <textarea placeholder="Your review here..." v-model="body"></textarea><br>
 
+      <OptionHeader>What session did you take the course?</OptionHeader>
+      <Selector :items="sessions" v-model="session" :placeholder="'Select a session..'"/>
```

Comment:
This needs some improvement imo;
- Should show in reverse order (later sessions at the top)
- Should only show up to current session
- Searchable maybe?

Reply:
Dropdown is good. I think don't worry about searching for now; with the other improvements it'll be useable